bindings/protocols/modbus/index.html

```diff
@@ -203,37 +203,37 @@ <h3>Form terms</h3>
+            <td><code>modv:pollingTime</code></td>
+            <td>Modbus TCP maximum polling rate. The Modbus specification does not define a maximum or minimum allowed polling rate, however specific implementations might introduce such limits. Defined as integer of milliseconds.</td>
             <td>optional</td>
-            <td><a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
 
         <tr>
         
 
         <tr>
-            <td><code>modv:mostSignificantWord</code></td>
-            <td>When <code>true</code>, it describes that the word order of the data in the Modbus message is the most significant word first (i.e., no word swapping). When <code>false</code>, it describes the least significant word first (i.e. word swapping)</td>
+            <td><code>modv:timeout</code></td>
+            <td>Modbus response maximum waiting time. Defines how much time the runtime should wait until it receives a reply from the device.</td>
             <td>optional</td>
-            <td><a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
 
         <tr>
         
 
         <tr>
-            <td><code>modv:pollingTime</code></td>
-            <td>Modbus TCP maximum polling rate. The Modbus specification does not define a maximum or minimum allowed polling rate, however specific implementations might introduce such limits. Defined as integer of milliseconds.</td>
+            <td><code>modv:mostSignificantByte</code></td>
+            <td>When <code>true</code>, it describes that the byte order of the data in the Modbus message is the most significant byte first (i.e., Big-Endian). When <code>false</code>, it describes the least significant byte first (i.e., Little-Endian).</td>
             <td>optional</td>
-            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
 
         <tr>
         
 
         <tr>
-            <td><code>modv:timeout</code></td>
-            <td>Modbus response maximum waiting time. Defines how much time the runtime should wait until it receives a reply from the device.</td>
+            <td><code>modv:mostSignificantWord</code></td>
+            <td>When <code>true</code>, it describes that the word order of the data in the Modbus message is the most significant word first (i.e., no word swapping). When <code>false</code>, it describes the least significant word first (i.e. word swapping)</td>
             <td>optional</td>
-            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
```

Comment:
Did `integer` disappear?

Reply:
No this is the type of mostSignificantWord . integer is the type of timeout which got moved above

Reply:
Ahh, I see